lib/mercadopago/collection.rb

```diff
@@ -38,6 +39,7 @@ def self.notification_preapproval(access_token, preapproval_id)
     # - access_token: an access_token of the MercadoPago account associated with the payment to be checked.
     # - search_hash: querystring in hash format.
     # - sandbox: whether or not the sandbox mode should be activated.
+    # - auto_paginate: wheter or not the result should be automaticaly paginated to return all payments
```

Comment:
@cleicar I think it's spelled "whether" - Please correct that. 

@mauro-oto @cleicar I think the library would be easier to use if the default of `auto_paginate` were `true`. Why did we make the default `false`?

Reply:
@etagwerker @cleicar users of the library expect this old method to return a limited amount of records, and they are (or rather, should be) using their own pagination technique around it. If we made it the default, there may or may not be bugs in their app code stemming from this library change. 

I think that making the default value for `auto_paginate` `true` is a good idea for the next major version release, since it's a feature many could benefit from, but not for a minor or patch release.